pandas/tests/indexes/interval/test_interval.py

```diff
@@ -906,6 +906,42 @@ def test_pickle_round_trip_closed(self, closed):
         tm.assert_index_equal(result, idx)
 
 
+def test_from_strings():
+    """Test the IntervalIndex.from_strings class method."""
+    # Create (expected, string_repr) tuples for test-cases.
+    test_cases = [
```

Comment:
can you parameterize the test function
include the link to the github issue as a comment

Reply:
Added both in 162045a